gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
+    layouts = {
```

Comment:
Being (ultimately) based on `CompositeOutputDevice`, the user is free to set the `.value` property (tuple of bools) to whatever combination they see fit - we don't restrict what combinations they can use. And as mentioned before, I'm not sure that having a property-getter that throws Exceptions is a good idea :-/